events/l6_christian.txt

```diff
@@ -160,8 +161,8 @@ l6_christian.3 = {
 		NOR = {
 			religion = manichaeism
 			religion = christianity
-			tag = JUD
-			tag = SMA
+			#tag = JUD
```

Comment:
Why not remove instead of commenting out?

Reply:
I comment out while testing to track changes in the file itself if I'm not entirely comfortable deleting something. At this point, I can remove.

Thanks!